Edit doc overall based on PyAnsys doc style guidelines (#655)
pyansys/pyfluent: doc/source/user_guide/models.rst

Defining models
===============
PyFluent supports defining models using :ref:`ref_solver_tui_commands` and :ref:`ref_settings`.

Using solver TUI commands
-------------------------
The following examples show how you define models using :ref:`ref_solver_tui_commands`.

Enabling the energy model
~~~~~~~~~~~~~~~~~~~~~~~~~

> The ``define.models.energy`` TUI enables and disables the energy model.

Author: ```suggestion
The ``define.models.energy`` TUI command enables and disables the energy model.
```

    session.solver.tui.file.read_case(case_file_name='file.cas.h5')
    session.solver.tui.define.models.energy('yes', 'no', 'no', 'no', 'yes')

Enabling the viscous model
~~~~~~~~~~~~~~~~~~~~~~~~~~

> The ``define.models.viscous.laminar`` TUI enables and disables the laminar flow model.

Author: ```suggestion
The ``define.models.viscous.laminar`` TUI command enables and disables the laminar flow model.
```

The ``define.models.viscous.kw_sst`` TUI enables and disables the SST-kw turbulence model.

> The ``define.models.viscous.ke_standard`` TUI enables and ddisables the standard-ke

Author: ```suggestion
The ``define.models.viscous.ke_standard`` TUI command enables and disables the standard-ke
```

    session.solver.tui.define.models.viscous.kw_sst('yes')
    session.solver.tui.define.models.viscous.ke_standard('yes')

Enabling the radiation model
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The ``define.models.radiation`` TUI provides options for selecting different radiation models.

Author: ```suggestion
The ``define.models.radiation`` TUI command provides options for selecting different radiation models.
```

.. code:: python

    session.solver.tui.define.models.radiation.s2s('yes')
    session.solver.tui.define.models.radiation.p1('yes')

Enabling the multiphase model
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The ``define.models.multiphase`` TUI provides options for selecting different multiphase models.

Author: ```suggestion
The ``define.models.multiphase`` TUI command provides options for selecting different multiphase models.
```

    session.solver.tui.define.models.multiphase.model('mixture')
    session.solver.tui.define.models.multiphase.model('wetsteam')

> Uing settings objects

Author: ```suggestion
Using settings objects
```
